### PR TITLE
Small apperance tweaks for diagnostic tab

### PR DESCRIPTION
**Pull request by metalice**
## 📝 Description

> Add a brief description

## 🎥 Demo

image

### CODE REVIEW

File: src/views/virtualmachines/details/tabs/diagnostic/virtual-machine-diagnostic-tab.scss [hunk 1]
```diff
@@ -2,3 +2,17 @@
   display: flex;
   justify-content: space-between;
 }
+
+.VirtulMachineDiagnosticTabRow--expended {
```
Comment (pcbailey) [inline src/views/virtualmachines/details/tabs/diagnostic/virtual-machine-diagnostic-tab.scss:6]: You have a couple of typos in the class name: 'Virtul' -> 'Virtual' and 'expended' -> 'expanded'

File: src/views/virtualmachines/details/tabs/diagnostic/virtual-machine-diagnostic-tab.scss [hunk 2]
```diff
@@ -2,3 +2,17 @@
...
+  display: inline-flex;
+  align-items: baseline;
+  margin-left: var(--pf-global--spacer--3xl);
+  svg {
+    margin-right: var(--pf-global--spacer--sm);
+  }
+}
+.pf-c-table {
+  tr.VirtulMachineDiagnosticTabRow--row {
```
Comment (pcbailey) [inline src/views/virtualmachines/details/tabs/diagnostic/virtual-machine-diagnostic-tab.scss:15]: Typo in the class name: 'Virtul' -> 'Virtual'

Comment (metalice) [reply src/views/virtualmachines/details/tabs/diagnostic/virtual-machine-diagnostic-tab.scss]: thanks!

File: src/views/virtualmachines/details/tabs/diagnostic/virtual-machine-diagnostic-tab.scss (hunk 1 above)
Comment (metalice) [reply src/views/virtualmachines/details/tabs/diagnostic/virtual-machine-diagnostic-tab.scss]: thanks!

**Pull request merged**